W-20237397 Update sftp-connector-troubleshooting.adoc (#3065)
mulesoft/docs-connectors: sftp/2.6/modules/ROOT/pages/sftp-connector-troubleshooting.adoc

To get further information about enabling verbose logging for SDK components in your Mule application, see the xref:mule-sdk::troubleshooting.adoc[Troubleshooting SDK components] documentation.

== Enable Apache SSHD Library Logging

[NOTE]
====
Enabling TRACE level logging for `org.apache.sshd` generates a high volume of debug logs that can cause severe performance degradation, including high CPU usage, thread contention, and system-wide delays. 
====

. Access Anypoint Studio and navigate to the *Package Explorer* view.
. Open your application's project name.
. Open the `src/main/resources` path folder.
. Open the `log4j2.xml` file inside the folder.
. Add this line to enable Apache SSHD library logging:
+
[source,xml,linenums]
----
    <AsyncLogger name="org.apache.sshd" level="TRACE"/>
----

== Enable SSL/TLS Handshake Debugging 

For diagnosing SSL/TLS handshake issues, add this argument to your run configuration:

`-M-Djavax.net.debug=ssl,handshake`

> This argument enables detailed SSL/TLS debugging output and should only be used when troubleshooting encryption-related connection issues.

Reviewer: ```suggestion
This argument enables detailed SSL/TLS debugging output. Use the argument only when troubleshooting encryption-related connection issues.
```
